charts/ever-gauzy-api/templates/deployment.yaml

```diff
@@ -60,7 +60,11 @@ spec:
             - name: SENTRY_POSTGRES_TRACKING_ENABLED
               value: {{ .Values.sentry.sentryPostgresTrackingEnabled | quote }}
             - name: DB_HOST
+            {{- if .Values.postgresql.global.external_db }}
+              value: {{ .Values.postgresql.global.postgresql.auth.username | quote }}
```

Comment:
for DB_HOST you want to use `username`??

Reply:
Nah it a typo didn't commit my last change.

Reply:
committed

Reply:
OK, I will merge, but what about DB_CERT? Not needed in AWS RDS or?